### ISSUE TITLE
[BP-2182] Can not cancel order with klarnakp payment method. 

### ISSUE THREAD

**Issue by kaplansin** [buckaroo-it/Magento2#641]
Afer selecting Klarnakp method we are redirecting to Payment Gateway page,
if we use Go Back button on this page, 
buckaroo/redirect/process controller called with brq_statuscode = 490
which is  STATUSCODE_FAILED 
order reservations are removed (adding positive reservation), quote is activated again
But order state remains as 'new' and , and we have seen that this endpoint called multiple times. 
on each cancel operation order state remains as new but magento is adding positive reservations, which is causing oversell

It is not clear why order cancellation has been disabled for klarnakp payment method. 

vendor/buckaroo/magento2/Model/Service/Order.php:224

```
 if (in_array($order->getPayment()->getMethodInstance()->buckarooPaymentMethodCode, ['klarnakp'])) {
                $this->logging->addDebug(__METHOD__ . '|25|');
                return true;
            }
```

**Comment by kaplansin** [comment]
Hi @Buckaroo-Rene  
do you have any update, this issue is effecting our salable quantity,
we are planning the remove this if condition, in this case order will be saved as 'Canceled'.

vendor/buckaroo/magento2/Model/Service/Order.php:224
```
 if (in_array($order->getPayment()->getMethodInstance()->buckarooPaymentMethodCode, ['klarnakp'])) {
                $this->logging->addDebug(__METHOD__ . '|25|');
                return true;
            }
```

**Comment by Buckaroo-Rene** [comment]
Hi @kaplansin ,

We are working towards a release and a fix for this issue will be included.
Currently we are in the testing phase, so I hope to do a release somewhere  next week (depending on the test results, of course).

**Comment by Buckaroo-Rene** [comment]
@kaplansin ,

We've just released version 1.44.0 of the Buckaroo Plugin.
A fix for this issue is included [[BP-2182]](https://github.com/buckaroo-it/Magento2/releases/tag/v1.44.0).
